circle.yml

```diff
@@ -17,6 +17,21 @@ jobs:
         command: |
           codespell --quiet-level=4 --ignore-words=./.codespell-whitelist
 
+  bumpversion:
+    docker:
+    docker:
+    - image: ethereum/cpp-build-env:10
+    steps:
+    - checkout
+    - run:
+        name: "Check bumpversion"
+        command: |
+          pip3 install bumpversion
+          export PATH="/home/builder/.local/bin:$PATH"
```

Comment:
Do it before `pip3 install`, otherwise `pip3` will complain.

Reply:
Changed.